src/palace/manager/api/axis.py

```diff
@@ -498,6 +507,12 @@ def fulfill(
             return fulfillment
         # If we made it to this point, the patron does not have this
         # book checked out.
+        self.log.warning(
+            "Unable to fulfill because there is no active loan.  See info statements below for details."
+        )
+        for statement in debug_info:
+            self.log.info(statement)
```

Comment:
Minor: Variable naming here is kind of deceiving, since you are logging a variable named `debug_info` at log level INFO, I would have assumed it would go into the DEBUG logs.